src/components/ui/Link/index.js

```diff
@@ -47,11 +49,10 @@ const Link = (props) => {
           {' '}
```

Comment:
I wonder if this should be removed in favour of some margins, or if it should at least be a `<span>` with the `text-decoration` and other styles removed. I agree that removing the underline from the external link indicator is nice, but I don't like the extra underline past the text here:

<img width="133" alt="Screenshot 2020-04-03 18 38 07" src="https://user-images.githubusercontent.com/90871/78389379-8a30dd80-75da-11ea-8699-2184e5a987e6.png">

I'd like it if we could remove that before merging.

Reply:
Good catch—I agree!

New version creates the space with some margin:
<img width="558" alt="Screenshot 2020-04-03 at 18 51 51" src="https://user-images.githubusercontent.com/376315/78390293-3a531600-75dc-11ea-84a4-0fd8f2b9540e.png">